pkg/capabilities/consensus/ocr3/datafeeds/feeds_aggregator.go

```diff
@@ -312,6 +321,14 @@ func ParseConfig(config values.Map) (aggregatorConfig, error) {
 			feedCfg.Deviation = dec
 			parsedConfig.Feeds[feedID] = feedCfg
 		}
+		if len(feedCfg.RemappedIDHex) > 2 {
```

Comment:
what is magical about 2? are you trying to check for a `0x` prefix? if so, string trim it explicitly.